Tasks/SonarQubePostTest/SonarQubePostTestImpl.ps1

```diff
@@ -0,0 +1,106 @@
+function InvokeMsBuildRunnerPostTest
```

Comment:
All this code appears as new because I renamed the file. However it was reviewed by Duncan today.